src/routes/index.js

```
@@ -23,5 +24,6 @@ router.use('/categories', category)
 router.use('/countries', country)
 router.use('/upload', uploadRoutes)
 router.use('/subjects', subject)
+app.use('/api/languages', languageRoutes);
```

ESLint

[error] 'app' is not defined.

(no-undef)